stories/Button/Button.story.jsx

```diff
@@ -47,7 +47,11 @@ const exampleTab = (
           </LinkTo>
           .
         </p>
+
```

Comment:
I'm putting here just because of the file, but it would be nice to add a option to remove the text and show only the icon btn in the Usage tab. What do you think?

Reply:
In this case, children prop receives a node or an array of nodes and Quantum Ui does not support edition for these props.